core/templates/dev/head/pages/exploration_editor/editor_tab/StateEditor.js

```diff
@@ -84,6 +84,97 @@ oppia.controller('StateEditor', [
   }
 ]);
 
+// A service which handles opening and closing the training modal used for both
+// unresolved answers and answers within the training data of a classifier.
+oppia.factory('trainingModalService', [
+  '$rootScope', '$modal', 'AlertsService',
```

Comment:
Hey @seanlip, one question, `TrainingModalService` has already been extracted out of the StateEditor.js and merged into oppia's develop branch. I am guessing this might have shown up when @lukiee555 was resolving the merge conflicts. I think having `trainingModalService` in StateEditor.js as a change of this branch would be a problem, right ? Would it be better to first sync this branch with upstream/develop and then make the required changes ?

Reply:
Yes, that is a problem -- it looks like the merge conflicts may have been incorrectly resolved. There are two ways to handle this:

- Start a new branch from HEAD of develop, and replicate changes there.
- Make changes to the existing PR by pushing additional commits until the "Files Changed" tab is exactly what you want it to be (no more and no less).

The second way may be quicker, but it's important to really have a strong understanding of what you're doing. Otherwise, the first way is safer.